src/aind_metadata_mapper/open_ephys/camstim_ephys_session.py

```diff
@@ -0,0 +1,366 @@
+                x=x, y=y, z=z, unit="micrometer"
+            ),
+            "",
+        )
+
+    def ephys_modules(self) -> list:
+        """
+        Return list of schema ephys modules for each available probe.
+        """
+        newscale_coords = npc_sessions.get_newscale_coordinates(
+            self.motor_locs_path
+        )
+
+        ephys_modules = []
+        for probe_letter in self.available_probes:
+            probe_name = f"probe{probe_letter}"
+            manipulator_coordinates, notes = self.manipulator_coords(
+                probe_name, newscale_coords
+            )
+
+            probe_module = session_schema.EphysModule(
+                assembly_name=probe_name.upper(),
+                arc_angle=0.0,
+                module_angle=0.0,
+                rotation_angle=0.0,
+                primary_targeted_structure="none",
+                ephys_probes=[
+                    session_schema.EphysProbeConfig(name=probe_name.upper())
+                ],
+                manipulator_coordinates=manipulator_coordinates,
+                notes=notes,
+            )
+            ephys_modules.append(probe_module)
+        return ephys_modules
+
+    def ephys_stream(self) -> session_schema.Stream:
+        """
+        Returns schema ephys datastream, including the list of ephys modules
+        and the ephys start and end times.
+        """
+        modality = aind_data_schema_models.modalities.Modality
+
+        probe_exp = r"(?<=[pP{1}]robe)[-_\s]*(?P<letter>[A-F]{1})(?![a-zA-Z])"
+
+        def extract_probe_letter(s):
+            """
+            Extracts probe letter from a string.
+            """
+            match = re.search(probe_exp, s)
+            if match:
+                return match.group("letter")
+
+        times = npc_ephys.get_ephys_timing_on_sync(
+            sync=self.sync_path, recording_dirs=[self.recording_dir]
+        )
+
+        ephys_timing_data = tuple(
+            timing
+            for timing in times
+            if (p := extract_probe_letter(timing.device.name)) is None
+            or p in self.available_probes
+        )
+
+        stream_first_time = min(
+            timing.start_time for timing in ephys_timing_data
+        )
+        stream_last_time = max(
+            timing.stop_time for timing in ephys_timing_data
+        )
+
+        return session_schema.Stream(
+            stream_start_time=self.session_start
+            + datetime.timedelta(seconds=stream_first_time),
+            stream_end_time=self.session_start
+            + datetime.timedelta(seconds=stream_last_time),
+            ephys_modules=self.ephys_modules(),
+            stick_microscopes=[],
+            stream_modalities=[modality.ECEPHYS],
+        )
+
+    def sync_stream(self) -> session_schema.Stream:
+        """
+        Returns schema behavior stream for the sync timing.
+        """
+        modality = aind_data_schema_models.modalities.Modality
```

Comment:
change import to: from aind_data_schema_models.modalities import Modality
and then you can use Modality.BEHAVIOR without this var